File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/schs/SchsSCH0201Q04PrartListItemInfo.java
package nta.med.data.model.ihis.schs;

public class SchsSCH0201Q04PrartListItemInfo {
	private String jundalTable;
	private String jundalPart;
	private String jundalPartName;
	
	public SchsSCH0201Q04PrartListItemInfo(String jundalTable,
			String jundalPart, String jundalPartName) {
		super();
		this.jundalTable = jundalTable;
		this.jundalPart = jundalPart;
		this.jundalPartName = jundalPartName;
	}
	
	public String getJundalTable() {
		return jundalTable;
	}
	public void setJundalTable(String jundalTable) {
		this.jundalTable = jundalTable;
	}
	public String getJundalPart() {
		return jundalPart;
	}
	public void setJundalPart(String jundalPart) {
		this.jundalPart = jundalPart;
	}
	public String getJundalPartName() {
		return jundalPartName;
	}
	public void setJundalPartName(String jundalPartName) {
		this.jundalPartName = jundalPartName;
	}
	
	

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Schs/SchsSCH0201Q04PrartListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Schs
{
    [Serializable]
	public class SchsSCH0201Q04PrartListItemInfo
	{
		private String _jundalTable;
		private String _jundalPart;
		private String _jundalPartName;

		public String JundalTable
		{
			get { return this._jundalTable; }
			set { this._jundalTable = value; }
		}

		public String JundalPart
		{
			get { return this._jundalPart; }
			set { this._jundalPart = value; }
		}

		public String JundalPartName
		{
			get { return this._jundalPartName; }
			set { this._jundalPartName = value; }
		}

		public SchsSCH0201Q04PrartListItemInfo() { }

		public SchsSCH0201Q04PrartListItemInfo(String jundalTable, String jundalPart, String jundalPartName)
		{
			this._jundalTable = jundalTable;
			this._jundalPart = jundalPart;
			this._jundalPartName = jundalPartName;
		}

	}
}